src/locale/pt-BR/dim.json

```diff
@@ -279,7 +279,7 @@
       "Vault": "Há muitos '{{itemtype}}' no {{store}}."
     },
     "Classified": "Este item é confidencial e não pode ser transferido no momento.",
-    "Classified2": "Item confidencial. A Bungie não fornece informações sobre este item. Este item ainda não é transferível.",
+    "Classified2": "Classified item. Bungie does not yet provide information about this item. Add notes to this item and use the \"notes:\" search filter to find it.",
```

Comment:
A bunch of these got reverted since the string changed a bit. Is there a way to preserve some of it?

Reply:
As far as translation is concerned the translation is no longer current, hence being removed. Like when we changed the copy on the logout button multiple times; otherwise it could be wrong information,